### ISSUE TITLE
Training material for AI (natural law processor)

### ISSUE THREAD

**Issue by drllau** [gemwise-invests/NLP-legal-corpus#1]
The basic functionality is for client to drop their employment contract and the legal workflow extracts out key information like
1. legal parties to the agreement;
2. the section which contains non-compete clauses;
3. other boilerplate like jurisdiction.

So I've got a NLP which can use statistical techniques to classify sentences. But any type of AI needs training set to become proficient so need to source the following from jurisdictions around world (preferable Anglo-Saxon). 
* NDA (deed, vs mutual), 
* MoU (optionally dual-language), and 
* Powers of Attorney (multiple jurisdiction);

For now just link to samples on replies and I'll clean them into a consistent CSV text format.

**Comment by drllau** [comment]
Using a consistent semantic vocabulary to describe the field variables ... use [Sali Legal matter specification standard](https://salilegal.org/sali3/docs/2020-02-02%20SALI%20LMSS%20Overview.pdf) as starting point

![image](https://user-images.githubusercontent.com/14944510/78465373-7646c200-7727-11ea-9ddf-ac981d738c94.png)

The point is to move away from purely textual comparison towards microformats and document reasoning based on parties, intent and purpose. I can then apply another AI technique call [Frames](https://en.wikipedia.org/wiki/Frame_(artificial_intelligence)) to allow adducing missing information to complete the document, and more importantly, continually update as circumstances change (successor, additional parties etc). In IT world this would be called [continuous integration](https://en.wikipedia.org/wiki/Continuous_integration) to assure legal consistency, especially when the law changes. I differ from [Legible](https://www.thoughtriver.com/how-it-works) in not parsing for granular properties but identifying blocks for rapid navigation. It should sit at interesection of drafting support and [consumer marketplace](https://www.legalgeek.co/startup-map/).

**Comment by dchunglawyer** [comment]
Hadn't heard of SALI before very innovative.
Here is a link to a few of my own precedents: https://cp.sync.com/dl/1f68ff630/bqdc5mhr-tyg2f7ns-ju6y2nqy-gxdhztdm
The MOU is not general purpose. It was for a blockchain client and I just scrubbed the party details.
The General Power of Attorney is a Queensland State form, and can be limited by including terms but is otherwise effectively unlimited.

**Comment by drllau** [comment]
Well, gov forms are good in that there's no copyright claimants ... what **direction I want the Limited Power of Attorney to go** is for ad-hoc partnerships ... this person would "represent" employees, suppliers or customers in wanting to take over the successful portions of an (most likely overseas) firm pulling out. Because of potential retaliation (against employees) there needs to be a degree of privacy which as you may expect, causes headaches in contractual privity. Hence the limited power of attorney allows a single point (or several people if master-limited partnership structure assumed) to legally bind the whole group (quasi-class action if a real-lawyer gets involved in the litigation loop). The other area is Alternative Dispute Resolution ... Thoughts?

**Comment by dchunglawyer** [comment]
I agree that is a valid concern. However, a power of attorney is not necessary. Only an agency agreement would be required, and in fact, it may be preferred as an agent has no need to disclose the entity for which they act, whereas an attorney executing on behalf of a principal would need to do so.

**Comment by drllau** [comment]
OK ... make it an agency agreement but how do you ensure decision making once you get more people? Can an agent sign NDA on behalf of principals (cf directors who bind all employees)? The NDA is used to cover all the subject material for spin-out, the MoU just outlines the process, and the agency/LPoA addresses who is party to the discussions and responsible for shepherding [signatures](https://www.lexology.com/library/detail.aspx?g=3c0bbc7c-7dc0-479b-b38f-1c06b48b5051)

**Comment by dchunglawyer** [comment]
The parties who constitute the Principal in the Principal-Agent relationship needs to be established from the beginning. 

An Agent signing on behalf of a Principal can legally bind the Principal. However, to bind the Principal's employees, contractors, etc., all you can do is require the Principal to ensure  that those parties comply. So, in other words, the liability still stops with the Principal.

The issue I see is that, if negotiations are successful, a new special purpose vehicle will likely need to be established to actually undertake the transaction, so the Principal will not actually be the one who becomes the purchaser/contracting party. But I suppose if signing the NDA and MOU is all that is required, then that's not a problem.

**Comment by drllau** [comment]
Are there conflicts of interest when the agent is already an employee (eg mid-level manager?)  know often there are anti-siphoning rules (can't work on own business using company time/resources) but we're in a transitional phase to discuss anything from becoming an independent channel partner/redistributor to management bailout (reverse acquisition). Also if the australian office is a branch or likely controlled entity, it may be best to talk directly with head office as counter party (or make that part of successor planning?)

On the other hand, if the agent is considered an trust-like position considering a multi-stakeholder arrangement rather than the whole business ending up in bankruptcy what safeguards should be in place? I'm going to try encoding your MoU using SALI's ontology and see how far that gets me.

**Comment by dchunglawyer** [comment]
There is a conflict, and there may be a technical breach of an employment contract. However, practically speaking, I think it is unlikely to be a problem given that the employees (or their appointed representative-agent) will likely be a person in a good position to float the idea with upper management, and they will be leveraging personal relationships to get the matter properly considered by the head company.

The main company would be most concerned with the employee/s leaving the company and breaching their restraints of trade and non-competes if a deal is not done. Where there is one employee representing a number of other employees as agent (who might remain anonymous in early negotiations) then there might be some pressure on the representative to disclose the identity of the others, but I don't know if there is any way to mitigate that issue merely with legal documentation. We could certainly alert customers to the issues and possibilities as part of the package we sell them.

**Comment by drllau** [comment]
This is the game-theory ... if the company goes kaput, then as one party does not legally exist, then the non-compete is voided (by definition no competition). I suspect some sort of release in order to negotiate in good faith. On the other side, retaliation for corporate disloyalty (jumping ship early) would be what holding employees back which is why the agency/LimitedPoA should preserve privacy if so long as abide by non-use/non-disclosure restrictions of NDA subject material (eg cost breakdowns in later part of due-diligence). Well, that's where the extra billable hours (20% cut) comes from :-) as you make $$$ from selling the rules and being engaged as [referee ](https://en.wikipedia.org/wiki/Non-cooperative_game_theory)(ADR)!

![image](https://user-images.githubusercontent.com/14944510/78616336-83d68600-78a6-11ea-9144-1ac84cfc415f.png) ![image](https://user-images.githubusercontent.com/14944510/78616461-dc0d8800-78a6-11ea-9d56-966dc95216ac.png)

Lose-lose is when cut back on employee hours, whilst business suffers overburden of fixed costs eating into margins. 
Win-Lose is if the business just fires employees
Lose-Win if company goes kaput and employees non-compete is voided and they do direct business

**Comment by dchunglawyer** [comment]
Found a bunch of MOUs, NDAs and Agency Agreements from various jurisdictions. It was easier to move them into my cloud storage account than to link to each one. I've also included a couple of my own additional precedent documents:

https://cp.sync.com/dl/1f68ff630/bqdc5mhr-tyg2f7ns-ju6y2nqy-gxdhztdm

I haven't gone through all of the documents, but the ones I have opened look generally okay. Some of them are for specific purposes so there may be content which is not relevant for our current purposes, but from an NLP processing perspective they may be useful. There is a lot more variation in the Agency Agreements and MOUs, as compared to the Employment Agreements and NDAs.

They are from various jurisdictions around the world, some from Australia.

**Comment by dchunglawyer** [comment]
Some more Agency Agreements here: https://contracts.onecle.com/type/273.shtml

**Comment by drllau** [comment]
Just finished the hackathons ... yes plural since I was mentoring the EU TheGlobalHack and ACS FlattenTheCurveHack ... I've also found multiple country NDAs (I'll try and move the ones I found to your drive. ) ... the structures somewhat similar with the only tricky thing is that the PURPOSE (of NDA) can either be in definitions, within first few paragraphs or on a separate clause. Because this is pretty variable, the AI is going to have fits trying to classify that
